### PR TITLE
Problem: Release script does not create testnet binaries on `release/v4-croeseid` branch

### PR DESCRIPTION
**Pull request by devashishdxt**
Solution: Changed the release script to create testnet binaries

### CODE REVIEW

File: .github/workflows/release.yml
Comment (tomtau) [inline .github/workflows/release.yml:33]: is this correct?

**Review by tomtau — COMMENTED** [review]
will this also build only the goleveldb binaries?

**Comment by tomtau** [comment]
arm64 linux may need a patch as well: https://github.com/crypto-org-chain/chain-main/blob/8c05c8bc1f5430618ba4ebe13fa185998c265dad/.github/workflows/release.yml#L71 ?

File: .github/workflows/release.yml
Comment (devashishdxt) [reply .github/workflows/release.yml:33]: Yes. The DB backend defaults to `goleveldb`.

**Comment by devashishdxt** [comment]
> arm64 linux may need a patch as well:
> 
> https://github.com/crypto-org-chain/chain-main/blob/8c05c8bc1f5430618ba4ebe13fa185998c265dad/.github/workflows/release.yml#L71
> 
> ?

Changed

**Review by tomtau — APPROVED** [review]

**Comment by devashishdxt** [comment]
I'll wait for @yihuang to review this before merging.

File: .github/workflows/release.yml
Comment (yihuang) [reply .github/workflows/release.yml:33]: The asset name should has the surfix too: `chain-main_4.0.0-alpha3-croeseid_Darwin_arm64_croeseid.tar.gz`

Comment (devashishdxt) [reply .github/workflows/release.yml:33]: @allthatjazzleo Can you check the final asset name? I remember you removed the `_croeseid` postfix in https://github.com/crypto-org-chain/chain-main/pull/776 and all the automation scripts assume that there's no `_croeseid` postfix.

Comment (allthatjazzleo) [reply .github/workflows/release.yml:33]: @yihuang @devashishdxt 
according to our previous convention, we don't have `testnet` suffix
https://github.com/crypto-org-chain/chain-main/releases/tag/v3.1.1-croeseid

Comment (devashishdxt) [reply .github/workflows/release.yml:33]: @allthatjazzleo Final generated assets will look like this: https://github.com/devashishdxt/chain-main/releases/tag/untagged-cab15c548893a1949350

Comment (allthatjazzleo) [reply .github/workflows/release.yml:33]: > @allthatjazzleo Can you check the final asset name? I remember you removed the `_croeseid` postfix in #776 and all the automation scripts assume that there's no `_croeseid` postfix.

yes coz croeseid is already part of the version tag

**Review by yihuang — APPROVED** [review]
LGTM

**Pull request merged**
